docs/user-guide/release-notes/2.2.x.rst

```diff
@@ -8,4 +8,9 @@ See the :fixedbugs:`2.2.0`.
 
 The docker rsync distributor has been improved to perform incremental publishes in cases where
 repositories have experienced only additive changes since the previous publish with the rsync
-distributor.
\ No newline at end of file
+distributor.
+
```

Comment:
if you write release notes for docker 2.2.x then this PR should be submitted against 2.2-dev branch and not master